code/modules/mob/living/simple_animal/abnormality/zayin/pile_of_mail.dm

```diff
@@ -0,0 +1,253 @@
+//ripped off of wall gazer, sorry devs
+
+/mob/living/simple_animal/hostile/abnormality/mailpile
+	name = "Pile of mail"
+	desc = "An abnormality that is a pile of stamped letters, none reaching their intended receiver."
+	icon = 'ModularTegustation/Teguicons/32x48.dmi'
+	icon_state = "mailbox"
+	maxHealth = 1
+	health = 1
+	threat_level = ZAYIN_LEVEL
+	work_chances = list(
+		ABNORMALITY_WORK_INSTINCT = list(30, 40, 50, 60, 70),
+		ABNORMALITY_WORK_INSIGHT = 6,
+		ABNORMALITY_WORK_ATTACHMENT = list(70, 60, 50, 40, 30),
+		ABNORMALITY_WORK_REPRESSION = 0,
+		)
+	pixel_x = 0
+	base_pixel_x = 0
+
+	work_damage_amount = 0
+	work_damage_type = WHITE_DAMAGE
+
+	gift_type =  /datum/ego_gifts/mail
+	gift_message = "An empty envelope slowly makes its way into your hands."
+
+	max_boxes = 12
+
+	ego_list = list(
+		/datum/ego_datum/weapon/letter_opener
+		)
+	abnormality_origin = ABNORMALITY_ORIGIN_LOBOTOMY
+	var/REPRESSCOUNT = 0 // V causes painmode
+	var/painmode = FALSE //this refers to papercut damage increase when 3rd repression is done
+	var/AFFECTEDCELL = list() // turf with funny
+	var/cooldown
+	var/cooldown_time = 30 SECONDS
+
+
+/mob/living/simple_animal/hostile/abnormality/mailpile/NeutralEffect(mob/living/carbon/human/user, work_type, pe)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/mailpile/FailureEffect(mob/living/carbon/human/user, work_type, pe)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/mailpile/ZeroQliphoth(mob/living/carbon/human/user)
+	return
+
+//papercut due to failed work
+/mob/living/simple_animal/hostile/abnormality/mailpile/WorktickFailure(mob/living/carbon/human/user)
+	if(painmode)
+		user.apply_damage(50, RED_DAMAGE, user.run_armor_check(null, BRUTE), spread_damage = FALSE)
```

Comment:
This deals 50 damage, per tick for 12 ticks. this means it can deal anywhere between 0-600 damage. You must understand that the maximum health of a player without armor is 230. While armor can reduce damage by up to 90%, HE armor only goes to about 50-40% maximum. Look at bottle of tears for some better numbers to work with.